Operation history fails for big response

```
Original Stack Trace:
2023-11-16T13:27:58+07:00		*__checkpoint ⇢ Body from POST http://localhost:8080/powerauth-java-server/rest/v3/operation/list [DefaultClientResponse]
2023-11-16T13:27:58+07:00	Error has been observed at the following site(s):
2023-11-16T13:27:58+07:00		Suppressed: reactor.core.publisher.FluxOnAssembly$OnAssemblyException: 
2023-11-16T13:27:58+07:00		at org.springframework.core.io.buffer.LimitedDataBufferList.raiseLimitException(LimitedDataBufferList.java:99) ~[spring-core-6.0.11.jar:6.0.11]
2023-11-16T13:27:58+07:00	Caused by: org.springframework.core.io.buffer.DataBufferLimitException: Exceeded limit on max bytes to buffer : 1048576
2023-11-16T13:27:58+07:00		... 129 common frames omitted
2023-11-16T13:27:58+07:00		at com.wultra.security.powerauth.rest.client.PowerAuthRestClient.callV3RestApi(PowerAuthRestClient.java:121) ~[powerauth-rest-client-spring-1.5.0.jar:1.5.0]
2023-11-16T13:27:58+07:00		at com.wultra.core.rest.client.base.DefaultRestClient.postObject(DefaultRestClient.java:429) ~[rest-client-base-1.7.0.jar:na]
2023-11-16T13:27:58+07:00		at com.wultra.core.rest.client.base.DefaultRestClient.post(DefaultRestClient.java:383) ~[rest-client-base-1.7.0.jar:na]
2023-11-16T13:27:58+07:00	Caused by:
```

Depends on https://github.com/wultra/powerauth-server/issues/1132

Has to be backported into 1.5.x

--- a/enrollment-server/src/main/java/com/wultra/app/enrollmentserver/impl/service/MobileTokenService.java
+++ b/enrollment-server/src/main/java/com/wultra/app/enrollmentserver/impl/service/MobileTokenService.java
@@ -109,15 +109,17 @@ public class MobileTokenService {
         final OperationListForUserRequest request = new OperationListForUserRequest();
         request.setUserId(userId);
         request.setApplications(List.of(applicationId));
+        request.setPageNumber(0);
+        request.setPageSize(OPERATION_LIST_LIMIT);
         final MultiValueMap<String, String> queryParams = httpCustomizationService.getQueryParams();
         final MultiValueMap<String, String> httpHeaders = httpCustomizationService.getHttpHeaders();
-        final com.wultra.security.powerauth.client.model.response.OperationListResponse pendingList =
+        final com.wultra.security.powerauth.client.model.response.OperationListResponse operations =
                 pendingOnly ?
                 powerAuthClient.operationPendingList(request, queryParams, httpHeaders) :
                 powerAuthClient.operationList(request, queryParams, httpHeaders);
 
         final OperationListResponse responseObject = new OperationListResponse();
-        for (OperationDetailResponse operationDetail: pendingList) {
+        for (OperationDetailResponse operationDetail: operations) {
             final String activationFlag = operationDetail.getActivationFlag();
             if (activationFlag == null || activationFlags.contains(activationFlag)) { // only return data if there is no flag, or if flag matches flags of activation
                 final Optional<OperationTemplateEntity> operationTemplate = operationTemplateService.findTemplate(operationDetail.getOperationType(), language);
@@ -127,10 +129,6 @@ public class MobileTokenService {
                 }
                 final Operation operation = mobileTokenConverter.convert(operationDetail, operationTemplate.get());
                 responseObject.add(operation);
-                if (responseObject.size() >= OPERATION_LIST_LIMIT) {
-                    logger.info("Reached the limit of operation list ({}) for user ID: {}", OPERATION_LIST_LIMIT, userId);
-                    break;
-                }
             }
         }
         return responseObject;

